src/main/java/com/example/nlern/controllers/MainController.java

```diff
@@ -1,5 +1,5 @@
 package com.example.nlern.controllers;
-
+/*
```

Comment:
It is not a good practice to leave whole classes commented out like that "for future". Either create an "empty" version of it, un-comment it to enable all endpoints, or delete it entirely for now and recreate it later

Reply:
Deleted